src/validators.ts

```diff
@@ -165,3 +165,10 @@ export function map<K, V>(
     return result;
   };
 }
+
+export function multiType<F, S>(
```

Comment:
Me attempting to add validator for `int | string[]`. :smile:  Feel free to tear it apart.

Reply:
Thank you, changed.